data/json/statistics.json

```diff
@@ -2004,5 +2004,81 @@
     "stat_type": "count",
     "event_transformation": "portal_storm_happened",
     "description": { "str_sp": "portal storms seen" }
+  },
+  {
+    "id": "avatar_enters_hospital",
+    "type": "event_transformation",
+    "event_transformation": "avatar_enters_oter_type",
+    "value_constraints": {
+      "oter_type_id": {
+        "equals_any": [
+          "oter_type_str_id",
+          [
+            "hospital_1",
+            "hospital_2",
+            "hospital_3",
+            "hospital_4",
+            "hospital_5",
+            "hospital_6",
+            "hospital_7",
+            "hospital_8",
+            "hospital_9",
+            "hospital_1_roof",
+            "hospital_2_roof",
+            "hospital_3_roof",
+            "hospital_4_roof",
+            "hospital_5_roof",
+            "hospital_6_roof",
+            "hospital_7_roof",
+            "hospital_8_roof",
+            "hospital_9_roof",
+            "urban_35_1",
+            "urban_35_2",
+            "urban_35_3",
+            "urban_35_4",
+            "urban_35_5",
+            "urban_35_6",
+            "urban_35_7",
+            "urban_35_8",
+            "urban_35_9",
+            "urban_35_10",
+            "urban_35_11",
+            "urban_35_12",
+            "urban_35_13",
+            "urban_35_14",
+            "urban_35_15",
+            "urban_35_16",
+            "urban_35_17",
+            "urban_35_18",
+            "urban_35_19",
+            "urban_35_20",
+            "urban_35_21",
+            "urban_35_22",
+            "urban_35_23",
+            "urban_35_24",
+            "urban_35_25",
+            "urban_35_26",
+            "urban_35_27",
+            "urban_35_28",
+            "urban_35_29",
+            "urban_35_30",
+            "urban_35_31",
+            "urban_35_32",
+            "urban_35_33",
+            "urban_35_34",
+            "urban_35_35"
+          ]
+        ]
+      }
+    }
+          ]
```

Comment:
Copy this entire file into https://jsonlint.com/ and then work through your brackets error here.  :)

Reply:
![image](https://user-images.githubusercontent.com/14862800/229673207-62dcc895-535e-43d5-9461-a506af3c9681.png)

![image](https://user-images.githubusercontent.com/14862800/229673475-4d095981-9391-4f5f-ae43-cbfa6c1c65a6.png)


JSON Lint shows it as valid. The entire Statistics.json copied and pasted. Should I take additional steps?